### PR TITLE
Implementation of the new workflow

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: auto_nag/escalation.py [hunk 1]
```diff
@@ -48,14 +53,22 @@ def __init__(self, who, people):
...
         m = NPLUS_PAT.match(self.who)
         if m:
             return self.people.get_nth_manager_mail(person, int(m.group(1)))
         if self.who == 'director':
             sup = self.people.get_director_mail(person)
         elif self.who == 'vp':
             sup = self.people.get_vp_mail(person)
+        elif self.who == 'self':
+            return self.people.get_moz_mail(person)
+        else:
```
Comment (sylvestre) [inline auto_nag/escalation.py]: please remove the else

File: auto_nag/scripts/workflow/p2_merge_day.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    def ignore_bug_summary(self):
+        return False
+
+    def get_bz_params(self, date):
+        params = {
+            'product': 'Core',
+            'resolution': '---',
+            'f1': 'component',
+            'o1': 'casesubstring',
+            'v1': 'Networking',
```
Comment (sylvestre) [inline auto_nag/scripts/workflow/p2_merge_day.py]: Can you move that in the configuration?
For now, it is just Networking but we will extend that to other components.

File: auto_nag/escalation.py (hunk 1 above)
Comment (calixteman) [reply auto_nag/escalation.py]: sorry but I can't

File: auto_nag/mail.py [hunk 3]
```diff
@@ -35,7 +42,9 @@ def send(
     # special = '<p><b>To: {}</b></p><p><b>Cc: {}</b></p>'.format(To, Cc)
     # i = Body.index('<body>') + len('<body>')
     # Body = Body[:i] + special + Body[i:]
-    # To = ['cdenizet@mozilla.com']
+    # From = 'cdenizet@mozilla.com'
```
Comment (sylvestre) [inline auto_nag/mail.py]: Maybe this could be added in the configuration
like test: true ?

**Pull request merged**